doc/nrf/ug_zigbee_configuring_zboss_traces.rst

```diff
@@ -0,0 +1,158 @@
+.. _ug_zigbee_configuring_zboss_traces:
+
+Configuring ZBOSS traces in |NCS|
+#################################
+
+.. contents::
+   :local:
+   :depth: 2
+
+The :ref:`nrfxlib:zboss` (ZBOSS stack) comes included in the |NCS| in a set of precompiled libraries, which can complicate the debugging process.
+To help with that, the ZBOSS stack can be configured to print trace logs that allow you to trace the stack behavior.
+This page describes how to enable and configure ZBOSS trace logs.
+
+Trace logs are printed in binary form and require access to stack source files, which are to be decoded into log messages.
+Enabling trace logs can help with the debugging process even if you cannot decode them (due to lack of access to stack source files).''
+This is because of the information they can provide about the stack behavior whenever an issue is found or reproduced.
+
+.. note::
+     ZBOSS trace logs are meant to be used for debugging ZBOSS based applications.
+     These libraries are not certified and should not be used for production firmware or end products.
+
+
+
+Requirements
+************
+
+To collect and view the log files, you will need the following:
+
+* Software that allows for reading serial ports.
+  For example, if you are using Windows you can use :ref:`putty`.
+
+* Correct COM port
+  One way to find the correct COM port is by setting the Kconfig option :kconfig:`CONFIG_USB_DEVICE_PRODUCT` (for example, :kconfig:`CONFIG_USB_DEVICE_PRODUCT="Zigbee sample"`)  to help identify the device in the system's :guilabel:`Device Manager`.
+
+* An additional USB cable if trace logs through USB are enabled (see :ref:`ug_zigbee_configuring_zboss_traces_using_usb`).
+
+
+.. rst-class:: numbered-step
+
+Switch to ZBOSS libraries with compiled-in trace logs
+*****************************************************
+
+Set the Kconfig option :kconfig:`CONFIG_ZIGBEE_ENABLE_TRACES` to switch to ZBOSS libraries with compiled-in trace logs.
+
+The ZBOSS stack comes in a precompiled form and trace logs are not compiled-in by default.
+An additional set of ZBOSS libraries are available in nrfxlib, which does have trace logs compiled-in.
```

Comment:
I believe "are" is the correct form, because it is describing the noun closest to it (set of **libraries**). I think this is correct, but I may be wrong...

Reply:
I trust with this one